src/main/java/VASSAL/counters/PropertyExporter.java

```diff
@@ -0,0 +1,20 @@
+package VASSAL.counters;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import VASSAL.build.module.properties.PropertyNameSource;
+import VASSAL.build.module.properties.PropertySource;
+
+public interface PropertyExporter extends PropertyNameSource, PropertySource {
+
+  default Map<String, Object> getProperties() {
```

Comment:
C++ devs should like this, it's almost like multiple inheritance.

Clean design would probably be unifying all three Property*-Interfaces into one.